src/vmecpp/_util.py

```diff
@@ -86,9 +88,16 @@ def indata_to_json(
         msg = f"{filename} does not exist."
         raise FileNotFoundError(msg)
 
-    indata_to_json_exe = Path(
-        package_root(), "cpp", "third_party", "indata2json", "indata2json"
+    # Get the path to the packaged indata2json executable.
+    # For virtual environments, package_root() points to the source files,
+    # but this method will return the correct install location e.g. in your virtual environment.
```

Comment:
I don't understand the comment, it seems to imply that package_root does not work with virtual envs, but it does..? Should this say:

```suggestion
    # For editable installs, package_root() points to the source files,
    # while importlib.metadata.files will return the correct install location in all cases.
```